packages/roosterjs-editor-plugins/lib/plugins/Entity/EntityFeatures.ts

```diff
@@ -156,7 +156,7 @@ function cacheGetNeighborEntityElement(
 
                 if (checkForSameLine) {
                     const block = editor.getBlockElementAtNode(pos.node);
-                    if (!block.contains(node)) {
+                    if (!block || !block.contains(node)) {
```

Comment:
Just want to sanity check that you intend to fail this check if block is falsy.

Reply:
Yes. If editor content is totally empty, not even an empty DIV there, block will be null. In that case we should return null since there's no entity at all.